client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice.jsx

```diff
@@ -136,14 +137,13 @@ class PendingGSuiteTosNotice extends React.PureComponent {
 			<>
 				<p>
 					{ translate(
-						'Your mailbox {{strong}}%(emails)s{{/strong}} is almost ready! Complete the setup to activate your new email address.',
-						'Your mailboxes {{strong}}%(emails)s{{/strong}} are almost ready! Complete the setup to activate your new email addresses.',
+						'Your new user {{strong}}%(emails)s{{/strong}} is ready! Complete the setup in order to activate it.',
+						'Your new users {{strong}}%(emails)s{{/strong}} are ready! Complete the setup in order to activate them.',
```

Comment:
We say in the transactional emails that the _users are ready_ so I've updated the copy to be more consistent here.

Reply:
I think the second sentence could be simplified as:
> Complete the setup to activate it/them.

The "in order" here makes the sentence feel complicated to me.

Reply:
Thanks for your feedback, I'll update all those strings as you suggest.